py-polars/polars/datatypes/constants.py

```diff
@@ -36,9 +36,6 @@
         Datetime("ms"),
         Datetime("us"),
         Datetime("ns"),
-        Datetime("ms", "*"),
-        Datetime("us", "*"),
-        Datetime("ns", "*"),
```

Comment:
These are not valid data types (`*` is not an existing timezone) and they messed up my parametric test. I removed them. We can rethink these data type groups when improving the parametric tests.